src/id3/v2/tag.rs

```diff
@@ -547,21 +607,54 @@ impl SplitAndMergeTag for ID3v2Tag {
 		fn split_pair(
 			content: &str,
 			tag: &mut Tag,
-			current_key: ItemKey,
+			number_key: ItemKey,
 			total_key: ItemKey,
 		) -> Option<()> {
```

Comment:
Using `Option<()>` for early returns is sometimes handy but also confusing. Better return nothing and use if-let-else statements for early returns.

Reply:
Please allow me to confirm the details.

`split_pair()` is called in the `match` statement as the [match guard](https://doc.rust-lang.org/book/ch18-03-pattern-syntax.html#extra-conditionals-with-match-guards). `if let` guards are experimental now.

https://github.com/rust-lang/rust/issues/51114

Reply:
Sorry, I wanted to referr to _let-else statements_ introduced with Rust 1.65. These should be convenient here and allow to avoid the final return of a meaningless result.

Reply:
Thank you for details. I will consider to use let-else.